fullscreen.bs

```diff
@@ -230,6 +238,8 @@ if all of the following are true, and false otherwise:
  <li><p><var>element</var>'s <a>node document</a> is <a>allowed to use</a> the "<code><a
  data-lt="fullscreen-feature">fullscreen</a></code>" feature.
  <!-- cross-process, recursive -->
+
+ <li><p><var>element</var>'s <a>popover visibility state</a> is <a for="popover visibility state">hidden</a>.
```

Comment:
@josepharhar there's one remaining problem here I'd like your thoughts on. The _element_ here could be any element, but only HTML elements have a popover visibility state. A type check is needed first. Do you already have type checks in HTML in a style you want to copy? If not, I'd suggest:

```suggestion
 <li><p><var>element</var> <a for=Element>namespace</a> is not the <a>HTML namespace</a> or <var>element</var>'s <a>popover visibility state</a> is <a for="popover visibility state">hidden</a>.
```